ynput/ayon-core
# Chore: Move Photoshop client code

## Changelog Description
Moved Photoshop client code next to server codebase.

## Additional information
Copied webserver tool from tools to photoshop client code. Also added dependency on `wsrpc_aiohttp` to Photoshop addon.

## Testing notes:
1. Create package of photoshop and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Photoshopintegration should work as before.
